sdk/src/main/java/co/omise/android/models/CardParam.kt

```diff
@@ -2,6 +2,12 @@ package co.omise.android.models
 
 import com.fasterxml.jackson.annotation.JsonProperty
 
+/**
+ * CardParam holds all the information about a Card that can be supplied to create
```

Comment:
This model doesn't seem to support new billing address fields: https://www.omise.co/tokens-api#create

Reply:
Great point, will need to be added either in this version or another in the near future 👌